### PR TITLE
update/bc-refactor

### PR DESCRIPTION
**Pull request by andrews-john**
No ticket, tech debt, refactoring bordercolors implementation

### CODE REVIEW

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import FA from 'react-fontawesome';
 import InteractiveElement from 'Components/InteractiveElement';
 import { formatDate } from 'utilities';
 import AgendaItemLegs from '../AgendaItemLegs';
-import { borderColors } from '../Constants';
+// import { borderColors } from '../Constants';
```
Comment (scott062) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:8]:
```suggestion

```

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 2]
```diff
@@ -38,12 +39,16 @@ const AgendaItemRow = props => {
       }
       {
         !isCreate &&
-        <div className="ai-history-row" style={{ borderLeftColor: borderColor }}>
+        <div className={`ai-history-row agenda-border--${agendaStatus} `}>
           <div className="ai-history-status">
-            <div className="status-tag" style={{ backgroundColor: borderColor }}>
+            <div className={`status-tag agenda-status--${agendaStatus}`}>
+              {/* </div> <div className="status-tag" style={{ backgroundColor: borderColor }}> */}
+              {/* <div className={`remarks-pill remark-category--${remark.rc_code}`}> */}
```
Comment (scott062) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:46]:
```suggestion

```

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 3]
```diff
@@ -38,12 +39,16 @@ const AgendaItemRow = props => {
...
               {get(agenda, 'status_full') || 'Default'}
             </div>
-            <div className="poly-slash" style={{ backgroundColor: borderColor, color: borderColor }} >_</div>
+            <div className={`poly-slash agenda-status--${agendaStatus}`}>_</div>
+            {/* <div className="poly-slash" style={{
+            backgroundColor: borderColor, color: borderColor }} >_</div> */}
```
Comment (scott062) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:51]:
```suggestion

```

File: src/sass/_panelMeetingAgendas.scss [hunk 4]
```diff
@@ -105,6 +104,7 @@
             font-size: medium;
           }
         }
+
```
Comment (scott062) [inline src/sass/_panelMeetingAgendas.scss:107]:
```suggestion

```

File: src/sass/_variables.scss [hunk 5]
```diff
@@ -235,7 +235,7 @@ $abl-gray-config: (
   "Updated On": 10,
   "Notes": 11
 );
-
+
```
Comment (scott062) [inline src/sass/_variables.scss:238]:
```suggestion

```

File: src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx [hunk 6]
```diff
@@ -5,7 +5,7 @@ import { clone, get, take, takeRight } from 'lodash';
 import { formatDate, shortenString } from 'utilities';
 import InteractiveElement from 'Components/InteractiveElement';
 import AgendaItemLegs from '../AgendaItemLegs';
-import { borderColors } from '../Constants';
+// import { borderColors } from '../Constants';
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx:8]: 🧼

File: src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx [hunk 7]
```diff
@@ -51,8 +51,7 @@ const AgendaItemCard = props => {
 
   // eslint-disable-next-line no-console
   const editAI = () => { console.log('placeholder create AI'); };
-  const status_full = get(agenda, 'status_full') || 'Default';
-  const borderColor = borderColors[status_full];
+  const agendaStatus = get(agenda, 'status_short') || 'Default';
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemCard/AgendaItemCard.jsx:54]: 🙌

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 8]
```diff
@@ -20,8 +19,9 @@ const AgendaItemRow = props => {
   const perdet$ = perdet || get(agenda, 'perdet');
 
   // eslint-disable-next-line no-console
+  console.log('agenda!: ', agenda);
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:22]: 🧼

File: src/sass/_borderColor.scss [hunk 9]
```diff
@@ -0,0 +1,10 @@
+@each $remark-id, $color in $borderColor {
+    .agenda-status--#{$remark-id} {
+        background-color: $color; 
+        color: $color;
+    }
+    .agenda-border--#{$remark-id} {
+        border-left: 15px solid $color;
+    }
+}
```
Comment (elizabeth-jimenez) [inline src/sass/_borderColor.scss:9]: 😍

File: src/sass/_variables.scss [hunk 10]
```diff
@@ -235,7 +235,6 @@ $abl-gray-config: (
   "Updated On": 10,
   "Notes": 11
 );
-
```
Comment (elizabeth-jimenez) [inline src/sass/_variables.scss:238]: let's add this one back in

File: src/sass/_variables.scss [hunk 11]
```diff
@@ -258,6 +257,24 @@ $remarks: (
   "W": $color-cornflower-blue, // need to update this one
 );
 
+//**
+//* Border Colors
+//*
+$borderColor:(
```
Comment (elizabeth-jimenez) [inline src/sass/_variables.scss:263]: 🙌

File: src/sass/_agendaItemHistory.scss [hunk 12]
```diff
@@ -108,7 +105,6 @@
 
     .ai-history-card {
       background-color: $color-white;
-      border-left: 10px solid $color-black;
```
Comment (elizabeth-jimenez) [inline src/sass/_agendaItemHistory.scss:111]: notice the difference in border width for card vs row; we'll want to address this

File: src/sass/_borderColor.scss [hunk 13]
```diff
@@ -0,0 +1,10 @@
+@each $remark-id, $color in $borderColor {
+    .agenda-status--#{$remark-id} {
```
Comment (elizabeth-jimenez) [inline src/sass/_borderColor.scss:2]: instead of `agenda-status` let's call this something like `agenda-tag`, that way it is more clear what it is for

**Review by elizabeth-jimenez — DISMISSED** [review]
in _borderColor.scss, define a new class to handle row vs card. so instead of just `.agenda-border--#{$remark-id}` you'll have something like `.agenda-border-row--#{$remark-id}` and `.agenda-border-card--#{$remark-id}`. Update other files, where needed, to use new classes.

**Comment by elizabeth-jimenez** [comment]
Just a few changes, but otherwise good-to-go!

**Comment by scott062** [comment]
@andrews-john Just need to get all the checks passing. @pveland621 will help you

**Review by scott062 — APPROVED** [review]
🐗 📁  (boar - dir "border") 

👍

**Pull request merged**